ports.mk

```diff
@@ -172,6 +300,13 @@ iphone: $(OBJS)
 		-framework CoreFoundation -framework QuartzCore -framework Foundation \
 		-framework AudioToolbox -framework CoreAudio -lobjc -lz
 
+ios7: $(OBJS)
```

Comment:
Since this is the same as `iphone` I suggest to use `iphone` for simplicity reasons.